neofetch

```diff
@@ -93,6 +93,9 @@ get_distro() {
                     "tiny") distro="${distro//version*}" ;;
                 esac
 
+            elif type -p tazpkg >/dev/null; then
+                distro="SliTaz $(< /etc/slitaz-release)"
```

Comment:
Oops. Didn't see the second screenshot. Was focused on 'strata' so I thought it was a Bedrock Linux instance.

While we're at that, how would neofetch work in a Bedrock Linux instance...